internal/services/dns/dns_zone_resource.go

```diff
@@ -87,7 +87,7 @@ func resourceDnsZone() *pluginsdk.Resource {
 
 						"host_name": {
 							Type:         pluginsdk.TypeString,
-							Required:     true,
+							Optional:     true,
```

Comment:
Can we also reflect this change in the document?

Reply:
yes, I had a look. At first, I thought that there was some automated generation of the docs.